internal/pkg/service/common/task/orchestrator/orchestrator_test.go

```diff
@@ -110,8 +108,6 @@ func TestOrchestrator(t *testing.T) {
 		return d1.DebugLogger().CompareJSONMessages(`{"level":"debug","message":"not assigned%s"}`) == nil
 	}, 5*time.Second, 10*time.Millisecond, "timeout")
 
-	cancel()
-	wg.Wait()
 	d1.Process().Shutdown(ctx, errors.New("bye bye 1"))
 	d1.Process().WaitForShutdown()
 	d2.Process().Shutdown(ctx, errors.New("bye bye 2"))
```

Comment:
Context cancellation before the etcd client shutdown caused warnings from etcd client.